pkg/recordbot/recorder_test.go

```diff
@@ -41,65 +41,65 @@ func cleanupTestRecorder() {
 func TestCanCreateH264Writer(t *testing.T) {
 	mimeType := webrtc.MimeTypeH264
 	writer, err := createMediaWriter("test.mp4", mimeType)
-	assert.NotNil(t, writer)
-	assert.NoError(t, err)
+	require.NotNil(t, writer)
```

Comment:
`require` in testify works like `assert` in other languages. where it halts tests upon the first require failure.